tests/it/runtime/dub.d

```diff
@@ -18,7 +18,12 @@ unittest {
         writelnUt("\n\nReggae output:\n\n", runReggae("-b", "ninja").lines.join("\n"), "-----\n");
         shouldExist("reggaefile.d");
         auto output = ninja.shouldExecuteOk;
-        output.shouldContain("-debug -g");
+
+        version(Windows) {
+            // args in response file
```

Comment:
Shouldn't we test that the args are actually in the response file on Windows?

Reply:
That would IMO be a PITA, even require to run ninja with `-d keeprsp` to keep the input response file. `tests/ut/ninja.d` tests that `rspfile_contents` contains `$flags` (and all the other compiler args in the normal cmdline), and the enabled output check on Posix makes sure the expected flags make it into the normal cmdline.